README.md

```diff
@@ -34,6 +34,23 @@
     - minimal dependencies (for researches): **pipenv install**
 - go to src/corona_hakab_model folder
     - run: **python main.py**
+    
+## Creating a Constants JSON
+You can modify the constants for the models including the total execution steps, infection rates, medical state machine,
+detection test parameters and policy changes via a JSON file. An example can be found at 
+`tests/simulation_sanity_test/default_consts.json`.
+
+You can test your JSON file against the JSON schema for the
+constants, this file is found at `src/corona_hakab_model/consts-schema.json`.
+
+To do so, for those without PyCharm:
+1. Open `https://www.jsonschemavalidator.net` in your web browser.
```

Comment:
What you do if u are offline? (or god forbid in mamdas)

Reply:
PyCharm has a built in JSON schema validator:
https://www.jetbrains.com/help/idea/json.html#